packages/atauth/CMakeLists.txt

```diff
@@ -0,0 +1,146 @@
+# Configurable options
+option(ATAUTH_BUILD_TESTS "Build tests for atauth" OFF)
+
```

Comment:
Since atauth is both a library and produces the at_activate executable, there needs to be an option which by default doesn't generate the executable.